include/logging/log_backend.h

```diff
@@ -94,7 +95,25 @@ static inline void log_backend_put(const struct log_backend *const backend,
 }
 
 /**
- * @brief Function for reconfiguring backend to panic mode.
+ * @brief Notify backend about dropped log messages.
+ *
+ * Function is optional.
```

Comment:
To what exactly this "optional" note applies? This is docstring for log_backend_dropped(), it is not optional and should be called properly by the logging core.